models/ai87net-effnetv2.py

```diff
@@ -0,0 +1,92 @@
+###################################################################################################
+#
+# Copyright (C) Maxim Integrated Products, Inc. All Rights Reserved.
+#
+# Maxim Integrated Products, Inc. Default Copyright Notice:
+# https://www.maximintegrated.com/en/aboutus/legal/copyrights.html
+#
+###################################################################################################
+"""
+Cifar-100 Efficient net v.2
+"""
+import torch.nn as nn
+import ai8x
+
+class AI87EfficientNetV2(nn.Module):
+    """
+    EfficientNet v2 Model
+    """
+    def __init__(
+            self,
+            num_classes=100,
+            num_channels=3,
+            dimensions=(32, 32),  # pylint: disable=unused-argument
+            bias=False,
+            **kwargs
+    ):
+        super().__init__()
+
+        # Stem Layer
+        self.conv_stem = ai8x.FusedMaxPoolConv2dBNReLU(num_channels, 32, 3, pool_size=2, pool_stride=2, stride=1, batchnorm='Affine', padding=1, bias=bias,
+                                            eps=1e-03, momentum=0.01, **kwargs)
+        # Series of MBConv blocks
+        self.mb_conv1 = ai8x.MBConvBlock(32, 16, 3, stride=1, padding=0, bias=bias, se_ratio=None, expand_ratio=1,
```

Comment:
padding argument is not used in MBConvBlock, should we consider removing it from init not to create confusion?

Reply:
Sure